docs/src/main/sphinx/installation/cli.rst

```diff
@@ -13,28 +13,182 @@ Requirements
+authentication method is :ref:`cli-username-password-auth`.
+
+As part of starting the CLI, you can set the default catalog and schema. This
+allows you to query tables directly without specifying catalog and schema.
 
 .. code-block:: text
 
-    ./trino --server localhost:8080 --catalog hive --schema default
+  ./trino --server http://trino.example.com:8080 --catalog tpch --schema tiny
+
+  trino:tiny> SHOW TABLES;
 
-Run the CLI with the ``--help`` option to see the available options.
+    Table
+  ----------
+  customer
+  lineitem
+  nation
+  orders
+  part
+  partsupp
+  region
+  supplier
+  (8 rows)
 
-The CLI uses the HTTP protocol and the
-:doc:`Trino client REST API </develop/client-protocol>` to communicate
-with Trino.
+You can also set the default catalog and schema with the :doc:`/sql/use`
+statement.
+
+.. code-block:: text
+
+  trino> USE tpch.tiny;
+  USE
+  trino:tiny>
+
+Many other options are available to further configure how to start the CLI in
+interactive mode:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--catalog``
+    - Sets the default catalog. You can change the default catalog and schema
+      with :doc:`/sql/use`.
+  * - ``--client-info``
+    - Adds extra information about the client.
+  * - ``--client-request-timeout``
+    - Sets the duration for query processing, after which, the client request is
+      terminated. Defaults to ``2m``.
+  * - ``--client-tags``
+    - Adds extra tags information about the client and the CLI user. The tags
+      can be used as input for :doc:`/admin/resource-groups`.
+  * - ``--debug``
+    - Enables display of debug information during CLI usage for
+      :ref:`cli-troubleshooting`. Displays more information about query
+      processing statistics.
+  * - ``--disable-compression``
+    - Disables compression of query results.
+  * - ``--editing-mode``
+    - Sets key bindings in the CLI to be compatible with VI or
+      EMACS editors. Defaults to ``EMACS``.
+  * - ``--http-proxy``
+    - Configures the URL of the HTTP proxy to connect to Trino.
+  * - ``--network-logging``
+    - Configures the level of detail provided for network logging of the CLI.
+      Defaults to ``NONE``, other options are ``BASIC``, ``HEADERS``, or
+      ``BODY``.
+  * - ``--password``
+    - Prompts for a password. You can set the ``TRINO_PASSWORD`` environment
+      variable with the password value to avoid the prompt. For more
+      information, see :ref:`cli-username-password-auth`.
+  * - ``--schema``
+    - Sets the default schema. You can change the default catalog and schema
+      with :doc:`/sql/use`.
+  * - ``--server``
+    - The HTTP/HTTPS address and port of the Trino coordinator. The port must be
+      set to the port the Trino coordinator is listening for connections on.
+      Trino server location defaults to ``http://localhost:8080``.
+  * - ``--session``
+    - Sets one or multiple :ref:`session properties
```

Comment:
nit: `one or more` is what I've seen written most often.

Reply:
ok